tests/test_grid3d/test_grid_property_roff.py

```diff
@@ -155,6 +155,11 @@ def test_eq_transitivity(roff_param1, roff_param2, roff_param3):
         assert roff_param1 == roff_param3
 
 
```

Comment:
Not part of your changes now - but i want to mention it since i'm reviewing! Line 99 has assigned an array `values` with values from `code_values` - but this is never used. In the return on line 106 the `code_values` is used. Is this by purpose or is it something wrong here?

Reply:
Well spotted! That was indeed incorrect. We are moving away from these permissive constructors, but it must happen with a major version increment so it will take some time.